src/components/GraphDialog/GraphDialog.jsx

```diff
@@ -12,7 +12,13 @@ const GraphDialog = () => {
 
   if (event.open) {
     return (
-      <Dialog maxWidth={"xl"} open={true} onClose={handleClose}>
+      <NucDialog
+        open={true}
+        handleClose={handleClose}
+        title={"Graph"}
+        expandedDimensions={{ width: "75rem", height: "60rem" }}
+        minimizedDimensions={{ width: "65rem", height: "50rem" }}
```

Comment:
Can you change to "minimizedDimensions" and "maximizedDimensions", please? Also "maximized" in local storage

Reply:
✅